screens/Budget/components/missingBudget.tsx

```diff
@@ -52,17 +36,47 @@ const MissingBudget = (
         })
     })
 
-    function createCopy() {
-        alert("Your nearset budget is " + (closetBudget !== undefined ? MONTHS_ORDER[parseInt(closetBudget.split('-')[1])] : 'none'))
+    const [copyBudget] = useMutation<CopyBudgetMutation>(CopyBudgetDocument, {
+        variables: { passwordHash, month, year, id: closetBudgetId },
+        onError: (error => {
+            alert(error.message);
+        }),
+        onCompleted: ((response) => {
+            if (response.copyBudget.__typename == "BudgetSuccess") {
+                triggerRefetch()
+            }
+        })
+    })
+
+    useEffect(() => {
+        var closetBudget: string | undefined = ''
+        if (otherBudgets?.budgets.__typename == 'BudgetsSuccess') {
+            otherBudgets.budgets.budgets.forEach(ele => {
+                const monthString = MONTH_TO_NUM_STRING(ele.month.valueOf())
+                if (monthString === null || monthString === undefined) {
```

Comment:
monthString cannot be undefined because `MONTH_TO_NUM_STRING` has a type signature of `string | null`